adapters/repos/db/inverted/searcher.go

```diff
@@ -436,9 +437,7 @@ func (s *Searcher) extractInternalProp(propName string, propType schema.DataType
 	}
 }
 
-func (s *Searcher) extractIDProp(propName string, propType schema.DataType,
-	value interface{}, operator filters.Operator, class *models.Class,
-) (*propValuePair, error) {
+func (s *Searcher) extractIDProp(propName string, propType schema.DataType, value interface{}, operator filters.Operator, class *models.Class) (*propValuePair, error) {
```

Comment:
can we bring back the original casing here and in all other methods? with those line breaks that we had it was easier to read code bc the whole method signature was visible on my macbook without the need to scroll to the right